java/mtsj/pom.xml

```diff
@@ -161,6 +161,36 @@
           <reuseForks>true</reuseForks>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>jib-maven-plugin</artifactId>
+        <version>2.5.2</version>
+        <configuration>
+         <from>
+          <image>adoptopenjdk/openjdk11:jre-11.0.4_11-alpine</image>
+        </from>
+          <to>
+		   <image>${project.artifactId}:${project.version}</image>
+          </to>
+          <container>
+            <mainClass>com.devonfw.application.mtsj.SpringBootApp</mainClass>
+			<ports>
+				<port>8081</port>
+			</ports>
+          </container>
+        </configuration>
+		
+		 <executions>
+			<execution>
+			  <phase>package</phase>
+			  <goals>
+				<goal>dockerBuild</goal>
+			  </goals>
+			</execution>
+		</executions>
+		
+      </plugin>	  
+	  
```

Comment:
config looks good.
Could you align this with our coding-conventions and use `devonfw-ide` to properly format the XML (indendation with 2 spaces, remove empty lines).

Reply:
Found a problem here, the command _mvn package_ looks for docker demon, since the docker build has been added as part of the lifecycle. Mythaistar build is always expecting docker.

The other way to do this is, use _mvn compile jib:dockerBuild_ . In this case mts-core image is not ready to run,, giving _Noclassdefinitionfound_ error due to  bookingmanagement (this interface in API) is missing.  mts-server does not have any other modules.